src/pymedphys/dicom/_level2/header_tweaks.py

```diff
@@ -125,3 +126,38 @@ def adjust_rel_elec_density_cli(args):
         ds, adjustment_map, ignore_missing_structure=args.ignore_missing_structure)
 
     pydicom.write_file(args.output_file, new_ds)
+
+
+def RED_adjustment_map_from_structure_names(structure_names):
+    pattern = re.compile(r'^.*RED=(\d+\.?\d*)$')
```

Comment:
Number thingy? This is hard to read.

Reply:
In the end I left this as a regular expression. I understand that it is harder to read when regular expressions are not known. But in the end it seemed like it would be quite a bit easier to leave it using regex and just make it be able to handle more cases. See:

https://github.com/pymedphys/pymedphys/pull/93/files#r264970442